site/app/templates/calendar/Calendar.twig

```diff
@@ -66,6 +102,26 @@
                 <button type="button" onclick="openOptionsModal()" class="btn btn-primary">
                     <i class="fas fa-sliders"></i> Options
                 </button>
+                <select id="calendar-year-edit" onchange="changeYear(this);">
+                    {# create options based on current year #}
+                    {% for year in range(curr_year -100, curr_year +100) %}
+                        <option value="{{ year }}" {% if curr_year == year %} selected {% endif %}>{{ year }}</option>
+                    {% endfor %}
+                </select>
+                <select id="calendar-month-edit" onchange="changeMonth(this);">
```

Comment:
Could we somehow have a list of the months, and then use a loop. Its just quite a few lines, and would be cleaner to read if we used a loop to look through each month in a list of months